src/main/resources/catalog.json

```diff
@@ -240,6 +240,25 @@
         "name" : "userId",
         "required" : false
       } ]
+  }, {
+    "id" : 15,
+    "name" : "credentialsUsage",
```

Comment:
I'd suggest naming this `useCredentials` to stick with the naming scheme already established as `verbNoun`.